### PR TITLE
Adding features

### PR DESCRIPTION
**Pull request by managr**
Adding a list of features on the top.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,29 @@
...
+* [x] utility tasks (deadletter reload)
+* [x] localstack compatible
+* [x] rails support (preloads rails environment)
+* [ ] terraform pub/sub scripts
+* [ ] development mode (creates missing SNS/SQS on the fly)
+* [ ] alternative infrastructure (redis?, kafka?)
+* [ ] message compression
+* [ ] alternative serialization (protobuf)
+* [ ] subscription filtering
+* [ ] fifo queues support
```
Comment (ervamate) [inline README.md]:
```suggestion
* [ ] fifo queues support
* [ ] opentracing support
```

Comment (kbiedrzycki) [reply README.md]: Any idea how this could be achieved?

Comment (ervamate) [reply README.md]: Yes, it's quite simple. I might attempt to push a PR for that

Comment (ervamate) [reply README.md]: In short, we need to wrap publisher and receiver into OpenTracing blocks and define how to transmit the trace id (most likely via message attributes)

Comment (managr) [reply README.md]: Nice. But anyway please approve this PR if it’s fine for you. The feature list is meant to be open. Unmarked items are planned.

**Pull request merged**